spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/core/mapping/event/BeforeExecuteDmlEventTest.java

```diff
@@ -41,8 +41,8 @@ public void equalsHashcodeTest() {
 		assertThat(beforeExecuteDmlEvent).isNotEqualTo(null);
 		assertThat(beforeExecuteDmlEvent).isNotEqualTo(new Object());
 
-		assertThat(beforeExecuteDmlEvent.hashCode()).isEqualTo(beforeExecuteDmlEvent.hashCode());
-		assertThat(beforeExecuteDmlEvent.hashCode()).isEqualTo(beforeExecuteDmlEvent1.hashCode());
+		assertThat(beforeExecuteDmlEvent).hasSameHashCodeAs(beforeExecuteDmlEvent.hashCode());
+		assertThat(beforeExecuteDmlEvent).hasSameHashCodeAs(beforeExecuteDmlEvent1.hashCode());
```

Comment:
Same question -- remove `hashCode()` call in argument?

Reply:
Fixed.